src/plugins/getConfig.ts

```diff
@@ -1,80 +1,75 @@
 import axios from 'axios'
 
-export interface EnvConfigI {
-    $PAYMENT_PORTAL_URL?: string
-}
-
-// Detailed commentary on how this works @ github.com/bcgov/business-filings-ui/utils/fetch-config.ts
-export function getConfig (): Promise<EnvConfigI> {
-  return new Promise(async resolve => {
-    const { origin } = window.location
-    const { BASE_URL } = process.env
-
-    const response = await axios.get(origin + BASE_URL + 'config/configuration.json', {
-      headers: {
-        'Accept': 'application/json',
-        'ResponseType': 'application/json',
-        'Cache-Control': 'no-cache'
-      }
-    })
-
-    const baseURL = response.data[0]['URL'] + '/api/v1'
-
-    const paymentUrl = response.data[0]['PAYMENT_PORTAL_URL']
+export async function getConfig (): Promise<void> {
```

Comment:
This isn't as bad as it looks. I refactored it to:
- look like `getConfig()` in our other apps
- load things in the order they are in the configuration file
- renamed a few local variables